### ISSUE TITLE
Does POST /token use `extraQueryParameters` object?

### ISSUE THREAD

**Issue by thoo1** [AzureAD/microsoft-authentication-library-for-js#2785]
## Library

- [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
- [X] `@azure/msal-browser@2.6.1`
- [ ] `@azure/msal-node@1.x.x`
- [ ] `@azure/msal-react@1.x.x`
- [ ] `@azure/msal-angular@0.x.x`
- [ ] `@azure/msal-angular@1.x.x`
- [ ] `@azure/msal-angular@2.x.x`
- [ ] `@azure/msal-angularjs@1.x.x`

## Description

I am using `extraQueryParameters` argument in acquireTokenSilent. I need to use this to supply a safe_rollout query parameter to take advantage of AAD safe rollout via query param.
However I am noticing that POST /token does not pass along this query param.

Is this expected? Wouldn't that mean that the POST /token cannot take the new config changes that I've put into a safe rollout?
I have a debug trace where the call here https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/c6a7aada4a6644dc173b39208d5421178fa9ae78/lib/msal-common/src/client/RefreshTokenClient.ts#L138 does not use the extraQueryParameters object that is passed in as a part of `request`

Basically I think I am expecting the POST /token call to also add the extraQueryParameters object into the request body. I'm not 100% sure if this endpoint needs this safe rollout id, maybe you folks know better?

## Source

- [X] Internal (Microsoft)
- [ ] Customer request

**Comment by tnorling** [comment]
@thoo1 We do not currently support `extraQueryParameters` on the `/token` endpoint but this is something we've discussed as a potential future enhancement.
